src/whaleapi.py

```diff
@@ -1,19 +1,40 @@
 import json
 import requests
+import os
+import time
 
 class WhaleAPI:
-    api_key = 123
+    api_key = os.environ['API_KEY']
     url = "https://api.whale-alert.io/v1"
-    status = "status"
-    transactions = "transactions"
+    r = requests.get(url)
+    t = int(time.time())
```

Comment:
```suggestion
    req = requests.get(url) # req도 근데 좋은 네이밍은 아님
    now_timestamp = int(time.time())
```

변수 이름은 최대한 명확하게 써주시는 것이 좋습니다.

Reply:
이 점 꼭 유의하겠습니다.